collect_app/src/main/java/org/odk/collect/android/activities/AppListActivity.java

```diff
@@ -67,6 +69,8 @@
     protected String[] mSortingOptions;
 
     private boolean mIsSearchBoxShown;
+    private boolean mFirstTime = true;
+    private int moldSelected;
```

Comment:
“mold” (a hollow container used to give shape to molten or hot liquid material ...) or `mOld`?